test.sh

```diff
@@ -7,45 +7,21 @@ OS=${OS:="centos"}
 OS_VERSION=${OS_VERSION:="7"}
 PYTHON_VERSION=${PYTHON_VERSION:="2"}
 ACTION=${ACTION:="test"}
-IMAGE="$OS:$OS_VERSION"
-
-# Optionally specify repo and branch for osbs-client to test changes
-# which depend on osbs-client patches not yet available in upstream master
-OSBS_CLIENT_REPO=${OSBS_CLIENT_REPO:-https://github.com/containerbuildsystem/osbs-client}
-OSBS_CLIENT_BRANCH=${OSBS_CLIENT_BRANCH:-master}
+IMAGE=${IMAGE:="$OS:$OS_VERSION"}
+# The following might be referenced outside of `prep_linux`, i.e. before they're defined
+CONTAINER_NAME=${CONTAINER_NAME:="atomic-reactor-$OS-$OS_VERSION-py$PYTHON_VERSION"}
```

Comment:
This should be generated dynamic, i.e just `atomic-reactor-markdownlint-ruby` for makrdown lint. ALso fix container anme for bandit and pylint, it uses conflicting names